### PR TITLE
Fix typo in documentation

### PR DESCRIPTION
**Pull request by hrubi**

### CODE REVIEW

File: lib/elixir/lib/file.ex [hunk 1]
```diff
@@ -1542,7 +1542,7 @@ defmodule File do
   executes the given function and then reverts back
   to the previous path regardless of whether there is an exception.
 
-  The current working directory is temporarilly set for the BEAM globally. This
+  The current working directory is temporarily set for the BEAM globally. This
```
Comment (josevalim) [inline lib/elixir/lib/file.ex:1545]: Can you please fix the other one too? :)

Comment (hrubi) [reply lib/elixir/lib/file.ex:1545]: There's no other one. It was a copy paste, but I've added this one new word with wrong spelling :)

Comment (josevalim) [reply lib/elixir/lib/file.ex:1545]: Oh!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**